CMakeLists.txt

```diff
@@ -3,11 +3,11 @@
 # Use CMake to generate the build files for your platform
 #
 # This script supports some configuration through CMake arguments (-Dparam=val syntax):
-#   BUILD_TOOLS=1          sets up additional executables to be built along with the server (ProtoProxy, GrownBiomeGenVisualiser, MCADefrag)
-#   BUILD_UNSTABLE_TOOLS=1 sets up yet more executables to be built, these can be broken and generally are obsolete (GeneratorPerformanceTest)
-#   CROSSCOMPILE=1         marks the build as a crosscompile (doesn't include optimizations for current platform)
-#   DISABLE_SYSTEM_LUA=1   disables the use of system Lua interpreter; the tolua executable will be built and used instead. Incompatible with cross-compiling
-#   SELF_TEST=1            enables testing code to be built
+#   BUILD_TOOLS=1            sets up additional executables to be built along with the server (ProtoProxy, GrownBiomeGenVisualiser, MCADefrag)
+#   BUILD_UNSTABLE_TOOLS=1   sets up yet more executables to be built, these can be broken and generally are obsolete (GeneratorPerformanceTest)
+#   NO_NATIVE_OPTIMIZATION=1 disables CPU-specific optimisations, allows use on other CPUs of the same platform
```

Comment:
Well you don't need to take me as the absolute. After all, English is not my native language, so perhaps you can come up with an even better description?
How about this: `disables optimisations specific for the CPU in this machine. Without this flag, the executable need not work on any other CPU, even of the same platform` (I'm a bit confused about "may not" and "need not", which one is better here?)